boards/arm/stm32wb5mm_dk/stm32wb5mm_dk_defconfig

```diff
@@ -22,3 +22,4 @@ CONFIG_HW_STACK_PROTECTION=y
 
 # enable pin controller
 CONFIG_PINCTRL=y
+CONFIG_TLC59731=y
```

Comment:
Display can't be part of default configuration. If required, it could be made conditional on DISPLAY activation

Reply:
> Display can't be part of default configuration. If required, it could be made conditional on DISPLAY activation
Thanks, I will make it conditional

Reply:
Then, please use LED as condition (not DISPLAY of course)